packages/sdk-snippets/src/helpers/escape.ts

```diff
@@ -0,0 +1,123 @@
+export interface EscapeOptions {
```

Comment:
I didn't write this! It was a recent addition to HTTPSnippet in the past month for sanitizing weird characters in headers but it works great for what I need to do here.

https://github.com/Kong/httpsnippet/blob/master/src/helpers/escape.ts

Only changes I've made to it are to add the two functions at the bottom: `quotedString` and `escapeForObjectKey`